src/utils/daos.c

```diff
@@ -1399,10 +1404,14 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 			"	--group=ID         group who will own the container.\n"
 			"			   format: groupname@[domain]\n");
 			ALL_BUT_CONT_CREATE_OPTS_HELP();
+		} else if (strcmp(argv[3], "get-prop") == 0) {
+			fprintf(stream,
+			"container options (get-prop):\n"
```

Comment:
```suggestion
				"container options (get-prop):\n"
```

Reply:
I didn't use the tab in order to remain consistent with the other cases in this function. Looking at the other cases, I think this was to keep the text from exceeding that 80 character width rule.